dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -231,6 +231,15 @@ def _do_aggregations(
         for col in group_columns:
             collected_aggregations[None].append((col, col, "first"))
 
+        groupby_agg_options = context.schema[
+            context.schema_name
+        ].config.get_config_by_prefix("dask.groupby.aggregate")
+        # Update the config string to only include the actual param value
+        # i.e. dask.groupby.aggregate.split_out -> split_out
+        for config_key in list(groupby_agg_options.keys()):
+            groupby_agg_options[
+                config_key.rpartition(".")[2]
+            ] = groupby_agg_options.pop(config_key)
```

Comment:
Thinking about this more, would it make sense to consolidate this functionality in `get_config_by_prefix`? We would filter the config options by prefix and then return those options with the prefix stripped. I can leave suggestions for what I imagine this would look like:

```suggestion
```